Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -51,8 +51,8 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
+        'Geom' : 'Extended2017dev',
+        'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
@makortel the changes from the new HCAL geometry, tag, and customizations definitely are not statistically insignificant. You cannot use the new HCAL tag with pre11 GEN-SIM.

I first suggested making a separate testing workflow and Era for the HCAL changes, but @davidlange6 did not want me to do this. At this point, he should comment if he would accept this PR with such a (temporary) new workflow and Era before I make those changes.